docs/references/mqtt-namespace.md

```diff
@@ -1083,6 +1084,73 @@ Examples:
 }
 ```
 
+### Inventory Container Images
+
+#### List All Images
+
+Using the API exposed by Inventory-V1, the user can manage container images via external applications such as Everywhere Cloud. This operation lists all the images in the gateway.
```

Comment:
Better remove reference to EC, right @MMaiero  ? We can reference Eclipse Kapua.